src/libs/actions/IOU.ts

```diff
@@ -2363,6 +2783,64 @@ function requestMoney(
     }
 }
 
+function sendInvoice(
+    currentUserAccountID: number,
+    transaction: OnyxEntry<OnyxTypes.Transaction>,
+    invoiceChatReport?: OnyxEntry<OnyxTypes.Report>,
+    receiptFile?: Receipt,
+    policy?: OnyxEntry<OnyxTypes.Policy>,
+    policyTagList?: OnyxEntry<OnyxTypes.PolicyTagList>,
+    policyCategories?: OnyxEntry<OnyxTypes.PolicyCategories>,
```

Comment:
An object instead of many arguments?

Reply:
It's the same way in other places in this file, so I decided to keep it consistent. Let's see how further review will go, I'll do quick update if it's necessary